### PR TITLE
Feature/write file information to outbox repo gdev 280

### PR DESCRIPTION
**Pull request by MoritzHahn1337**
Title for squash and merge:
```
Subscribes to file_staged_for_download, updates database entry, if ncessary
```

Description for squash and merge:
```
Subscribes to file_staged_for_download
Checks, if the file really exists in outbox
Updates the file related information in the database, if necessary
```

### CODE REVIEW

**Comment by MoritzHahn1337** [comment]
Tests also still missing, do not merge

File: drs3/pubsub/subscribe.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        db_object_info = database.get_drs_object(message["file_id"])
+
+        # Check if file is in outbox
+        with ObjectStorage(config=CONFIG) as storage:
+            if storage.does_object_exist(
+                config.s3_outbox_bucket_id, message["file_id"]
+            ):
+
+                # Update information, in case something has changed
+                database.update_drs_object(db_object_info)
```
Comment (KerstenBreuer) [inline drs3/pubsub/subscribe.py:49]: Would the md5checksum, the size, and the registration date really change upon staging? (Once we are re-encrypting for an individual user that will be the case and we have to maintain infos on user-specific DRS-object copies.)

**Review by KerstenBreuer — CHANGES_REQUESTED** [review]
Initially, I thought we would have to keep a boolean flag in the database to check if a file is present in the storage or not. However, your implementation is right: We can get this info just by looking directly at the storage outbox. Maybe we have to re-think if we right now need the "file_staged_for_download" topic at all?

Let's discuss that tomorrow.

File: tests/unit/test_pubsub.py [hunk 2]
```diff
@@ -51,3 +59,26 @@ def test_publish_stage_request(amqp_fixture):  # noqa: F811
...
+        sources=[psql_fixture.config, s3_fixture.config, amqp_fixture.config]
+    )
+
+    # initialize upstream and downstream test services that will publish or receive
+    upstream_publisher = amqp_fixture.get_test_publisher(
+        topic_name=config.topic_name_file_staged,
+        message_schema=schemas.FILE_STAGED,
+    )
+
+    print(psql_fixture.existing_file_infos[0])
```
Comment (KerstenBreuer) [inline tests/unit/test_pubsub.py:76]:
```suggestion
```
Leftover debug statement?

**Review by KerstenBreuer — APPROVED** [review]
Looks all right

**Pull request merged**